test/functional/rpc_named_arguments.py

```diff
@@ -30,5 +30,16 @@ def run_test(self):
         assert_equal(node.echo(arg9=None), [None]*10)
         assert_equal(node.echo(arg0=0,arg3=3,arg9=9), [0] + [None]*2 + [3] + [None]*5 + [9])
 
+        # Test getblock verbosity
+        block = node.getblock(blockhash=h, verbosity=0)
+        assert(isinstance(block, str))
+
+        block = node.getblock(blockhash=h, verbosity=1)
+        assert(isinstance(block['tx'][0], str))
+
+        block = node.getblock(blockhash=h, verbosity=2)
+        assert('vin' in block['tx'][0])
+
+
```

Comment:
Is this functional test necessary? These are the only named arguments being tested

Reply:
I think it's necessary to test the different options, although it would perhaps make more sense to move it to rpc_blockchain.py